src/library/basetracktablemodel.cpp

```diff
@@ -471,6 +474,29 @@ QVariant BaseTrackTableModel::roleValue(
     case Qt::ToolTipRole:
         if (column == fieldIndex(ColumnCache::COLUMN_LIBRARYTABLE_COLOR)) {
             return mixxx::RgbColor::toQString(mixxx::RgbColor::fromQVariant(rawValue));
+        } else if (column == fieldIndex(ColumnCache::COLUMN_LIBRARYTABLE_COVERART)) {
+            TrackPointer pTrack = getTrack(index);
+            if (!pTrack) {
+                return kEmptyString;
+            }
+            CoverArtCache* pCache = CoverArtCache::instance();
+            QPixmap pixmap = QPixmap(kHeightOfCoverartTooltip,
+                    kHeightOfCoverartTooltip); // Height also used as default for the width, in assumption that covers are squares
+            pixmap = pCache->tryLoadCover(this,
+                    pTrack->getCoverInfoWithLocation(),
+                    kHeightOfCoverartTooltip,
+                    CoverArtCache::Loading::NoSignal);
+            if (pixmap.isNull()) {
+                // Cache miss -> Don't show a tooltip
+                return kEmptyString;
+            }
+            QByteArray data;
+            QBuffer buffer(&data);
+            pixmap.save(&buffer, "PPM"); // Most simple format, without compression effort
+            QString html = QString(
+                    "<img src='data:image/x-portable-pixmap;base64, %0'>")
+                                   .arg(QString(data.toBase64()));
```

Comment:
I imagine scrolling up & down the library and everytime the line underneath the cursor changes we create a new image?
can we cache the image? do we need to?
idk about how processing vs. memory recall effort do affect GUI performance though

Reply:
okay, in my os the tooltip fades out after some seconds anyway

Reply:
None of the library tooltips changes for me (Windows7), if the content below the mouse pointer changes.